apps/style/common.scss

```diff
@@ -1746,7 +1768,7 @@ a.WireframeButtons_active, div.WireframeButtons_active {
 
   #gameButtons {
     width: auto;
-    padding: 1% 5%;
+    padding: 4px 5%;
```

Comment:
The vertical padding was changed to be explicit, which is helpful for other height calculations in this area. Since `gameButton` is rendered at `400px` wide, `1%` is equal to `4px`